STM32Resources/BuildEnvironment_Setup/BuildEnvironment_Setup.adoc

```diff
@@ -0,0 +1,81 @@
+
+
+= *Setting up and Using an STM32 Build Environment on Linux*
+A guide for setting up a linux system to build projects for STM32
+
+
+This guide assumes you are starting with a Windows 64 bit system and intend to use a virtual machine in Oracle Virtualbox. I installed Linux Mint 18.3 If you already have a linux system set up, are using a Mac, or want to set it up a diffeent way figure out what parts of this guide you have already completed yourself and rejoin at a later step.
+
+== Install Virtualbox
+Download https://www.virtualbox.org/wiki/Downloads[Oracle Virtual Box]
+The latest version at writting was 5.2.4. Also download the VirtualBox Extension Pack from the same page.
+Install both.
+
+==== Setup a Virtual Machine
+Create a new virtual machine in Virtualbox.
```

Comment:
This step to me sounds too general, it can be more specific I think. 

```asciidoc
The first step is to create a new virtual machine in Virtualbox, this can be done by hitting the _New_ icon.
```